src/ontology/mondo-edit.obo

```diff
@@ -463641,6 +463697,48 @@ xref: DOID:0080619 {source="MONDO:equivalentTo"}
 is_a: MONDO:0000633 {source="DOID:0080619"} ! sensory organ benign neoplasm
 is_a: MONDO:0002409 {source="DOID:0080619"} ! auditory system disorder
 
+[Term]
+id: MONDO:0850154
+name: tongue carcinoma
+xref: DOID:0080641 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004631 ! tongue cancer
+is_a: MONDO:0004993 ! carcinoma
+
+[Term]
+id: MONDO:0850156
+name: b-lymphoblastic leukemia/lymphoma mll rearranged
+def: "A B-lymphoblastic leukemia/lymphoma that is characterized by the presence of lymphoblasts that carry a translocation between the KMT2A gene at 11q23.3 and another gene partner resulting in the production of a KMT2A related fusion protein." [DOID:0080644]
+xref: DOID:0080644 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004947 {source="DOID:0080644"} ! B-cell acute lymphoblastic leukemia
+
+[Term]
+id: MONDO:0850157
+name: b-lymphoblastic leukemia/lymphoma with etv6-runx1
+def: "A B-lymphoblastic leukemia/lymphoma that is characterized by the presence of lymphoblasts that carry a translocation between the TEL gene on chromosome 12 and the AML1 gene on chromosome 21, (p13.2;q22.1). It results in the production of the TEL-AML1 (ETV6-RUNX1) fusion protein." [DOID:0080645]
+xref: DOID:0080645 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004947 {source="DOID:0080645"} ! B-cell acute lymphoblastic leukemia
+
+[Term]
+id: MONDO:0850160
+name: b-lymphoblastic leukemia/lymphoma with il3-igh
```

Comment:
- B should be capital
- IL3 should be capital
- is 'igh' a typo? should be high?